chaos_kitten/brain/orchestrator.py

```
@@ -21,14 +34,159 @@ def __init__(self, config: dict[str, Any]) -> None:
+        console.print("[yellow]🧠 Planning attack strategies...[/yellow]")
+        planner = AttackPlanner(self.endpoints)
+        # planner.load_attack_profiles() # Stubbed in planner
+        
+        target_url = (
+            self.config.get("target", {}).get("base_url")
+            or self.config.get("api", {}).get("base_url")
+            or self.config.get("target")
+        )
+
+        if not target_url:
+             raise ValueError("No target URL provided (target.base_url or api.base_url)")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)